haiku-libs/sagebrush/sagebrush-0~git.recipe

```diff
@@ -56,11 +56,16 @@ INSTALL()
 
 TEST()
 {
-	gcc tools/Property\ Editor/*.cpp -Iinterface -Isupport -L. -lbe -lsagebrush
+	gcc "tools/Property Editor"/*.cpp -Iinterface -Isupport -L. -lbe -lsagebrush
+	LIBRARY_PATH=.:$LIBRARY_PATH ./PropertyWindow
```

Comment:
I think we should avoid putting `.` in `LIBRARY_PATH`.
If there are libs in the root directory of the build, then this would be better imo:
```diff
-	LIBRARY_PATH=.:$LIBRARY_PATH ./PropertyWindow
+	LIBRARY_PATH="$sourceDir${LIBRARY_PATH:+:$LIBRARY_PATH}" ./PropertyWindow
```
You can even insert
```
	export LIBRARY_PATH="$sourceDir${LIBRARY_PATH:+:$LIBRARY_PATH}"
```
at the beginning of the `TEST()` function and never have to worry about it anymore. In particular, there is no more need to add `../..` to the `LIBRARY_PATH` later.